funnel/static/translations/hi_IN/messages.json

```diff
@@ -1531,7 +1528,7 @@
       "Forgot password?": [null, "पासवर्ड भूल गए हैं?"],
       "By signing in, you agree to Hasgeek’s <a href=\"/about/policy/terms\" target=\"_blank\" rel=\"noopener\">terms of service</a> and <a href=\"/about/policy/privacy\" target=\"_blank\" rel=\"noopener\">privacy policy</a>.": [
         null,
-        "साइन इन करने पर, आप Hasgeek की <a href=\"/about/policy/terms\"target=\\_blank\\ rel=\noopener\\>सेवा की शर्तों</a> और <ahref=\\/about/policy/privacy\\ target=\\_blank\\ rel=\noopener\\>गोपनीयता निति</a> की सहमती देते हैं."
+        "साइन इन करने पर, आप Hasgeek की <a href=\"/about/policy/terms\"target=\"_blank\" rel=\noopener\">सेवा की शर्तों</a> और <ahref=\"/about/policy/privacy\" target=\"_blank\" rel=\noopener\">गोपनीयता निति</a> की सहमती देते हैं."
```

Comment:
`rel=\noopener\"` is missing a `"` in both instances. `…terms\"target=…` is missing a space.